snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -444,33 +444,36 @@
         }
     }
 
-    // Listen for clicks on links that point to 'about:accounts' and
-    // convert them showFirefoxAccounts Events.
-    document.addEventListener('click', function(event) {
+    // Listen for clicks on links, send metrics and handle
+    // about:account custom links.
+    var topSection = document.getElementById('topSection');
+    topSection.addEventListener('click', function(event) {
         var target = event.target;
-        while (target && target.tagName && target.tagName.toLowerCase() !== 'a') {
+        while (target.tagName && target.tagName.toLowerCase() !== 'a') {
+            // Do not track clicks outside topSection.
+            if (target.id === 'topSection') {
```

Comment:
Correct, only one element with id=topSection. What we do here is that we traverse the tree in reverse trying to find an `<a>`. We can start within `topSection` but because of `target=target.parentNode` get out of it and that's why we need this check, correct?